README.md

```diff
@@ -60,7 +60,19 @@ Once you have implemented a lower bound search for the start of the range, imple
 
 *How do you use binary search to find the lower bound of a number? How did you have to modify the binary search algorithm?*
 
+When looking for a specific value in a binary search the algorithm includes a break, for when that value is found. We are not sure to have the actual value, and as such the break condition is removed to make the algorithm run until we get as close to the value as possible where low=high
```

Comment:
With the right invariant (that you can probably write), you have that `low` is the point you are looking for when `low == high`.